samples/java/README.md

```diff
@@ -0,0 +1,18 @@
+
+How to run:
+
+__Intellij__
+
+* Open the project in IntelliJ
+* [Optional] Change the version number of indy to latest in pom.xml like:
+    ``` 
+    <dependency>
+       			<groupId>org.hyperledger</groupId>
+       			<artifactId>indy</artifactId>
+       			<version>1.0.0</version>
+     </dependency>
+     ```
+     
+* Click on Edit configuration and edit Main.config, add DYLD_LIBRARY_PATH=<path to indy dll/so/dylib>
```

Comment:
Name of DYLD_LIBRARY_PATH variable is platform specific. On Linux it will be LD_LIBRARY_PATH, on Windows just PATH. Also, it would note about execution pool in Docker.

Reply:
Done